.gitignore

```diff
@@ -1,5 +1,5 @@
 #ignore all bazel build files
 /bazel-*
 
-#ignore local editor files
```

Comment:
Please move gitignore into its own change set, since it's not related to the rest of this,

Reply:
yup makes sense!